src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -503,10 +601,6 @@ class SmartSearchBar extends React.Component {
     return [];
   };
 
-  onInputClick = () => {
```

Comment:
What happens when an item is clicked now?

Reply:
No change to `onClick`. I moved it to Line 326 so it is closer to the `keyUp`/`keyDown` methods.